FV3/conf/configure.fv3.gnu_docker_serialize

```diff
@@ -0,0 +1,136 @@
+
+############
+# commands #
+############
+FC = FC=gfortran mpifort
```

Comment:
We already discussed this, but it is hard to accept having another copy of this file with all the same flags just to add a couple... but I think you'd be more opposed to it than me. If you are ok with it, I can be as well.

Reply:
`configure.fv3.gnu_docker` itself is just a copy of `configure.fv3.gnu_linux` with just a couple flags changed.

Reply:
correction, *yet another :).